.github/workflows/release-tagger.yaml

```diff
@@ -0,0 +1,23 @@
+name: Tag Release
+on:
+  pull_request:
+    types: [closed]
```

Comment:
Is this going to run on every closed PR? Even if not accepted?

Reply:
I think you missed L12

Reply:
It will trigger - but won't run because of line 12 as @aidy Mentions, there's an if ensuring that it was merged and the originating branch was `release-XXX`